graylog2-server/src/test/java/org/graylog2/contentpacks/facades/DashboardV1FacadeTest.java

```diff
@@ -0,0 +1,187 @@
+        objectMapper.registerSubtypes(new NamedType(MessageListConfigDTO.class, MessageListConfigDTO.NAME));
+        objectMapper.registerSubtypes(new NamedType(LineVisualizationConfigDTO.class, LineVisualizationConfigDTO.NAME));
+        objectMapper.registerSubtypes(new NamedType(BarVisualizationConfigDTO.class, BarVisualizationConfigDTO.NAME));
+        objectMapper.registerSubtypes(new NamedType(NumberVisualizationConfigDTO.class, NumberVisualizationConfigDTO.NAME));
+        objectMapper.registerSubtypes(new NamedType(TimeHistogramConfigDTO.class, TimeHistogramConfigDTO.NAME));
+        objectMapper.registerSubtypes(new NamedType(ValueConfigDTO.class, ValueConfigDTO.NAME));
+        objectMapper.registerSubtypes(new NamedType(PivotSortConfig.class, PivotSortConfig.Type));
+        objectMapper.registerSubtypes(new NamedType(PivotEntity.class, PivotEntity.NAME));
+        objectMapper.registerSubtypes(new NamedType(PivotSort.class, PivotSort.Type));
+        objectMapper.registerSubtypes(new NamedType(OrFilter.class, OrFilter.NAME));
+        objectMapper.registerSubtypes(new NamedType(StreamFilter.class, StreamFilter.NAME));
+        objectMapper.registerSubtypes(new NamedType(QueryStringFilter.class, QueryStringFilter.NAME));
+        objectMapper.registerSubtypes(new NamedType(AutoIntervalDTO.class, AutoIntervalDTO.type));
+        searchDbService = new ViewFacadeTest.TestSearchDBService(mongoRule.getMongoConnection(),
+                new MongoJackObjectMapperProvider(objectMapper));
+        viewService = new ViewFacadeTest.TestViewService(mongoRule.getMongoConnection(),
+                new MongoJackObjectMapperProvider(objectMapper), null);
+        facade = new DashboardV1Facade(objectMapper, searchDbService, viewService);
+    }
+
+    @Test
+    @UsingDataSet(loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
+    public void shouldConvertAndInstallAOldDashboardAsNewView() throws Exception {
+        final URL resourceUrl = Resources.getResource(DashboardV1Facade.class, "content-pack-dashboard-v1.json");
+        final ContentPack contentPack = objectMapper.readValue(resourceUrl, ContentPack.class);
+        assertThat(contentPack).isInstanceOf(ContentPackV1.class);
+        final ContentPackV1 contentPackV1 = (ContentPackV1) contentPack;
+        final Entity entity = contentPackV1.entities().iterator().next();
+
+        final StreamImpl stream = new StreamImpl(Collections.emptyMap());
+        final Map<EntityDescriptor, Object> nativeEntities = new HashMap<>(1);
+        nativeEntities.put(EntityDescriptor.create("58b3d55a-51ad-4b3e-865c-85776016a151", ModelTypes.STREAM_V1), stream);
+
+        final NativeEntity<ViewDTO> nativeEntity = facade.createNativeEntity(entity,
+                ImmutableMap.of(), nativeEntities, "kmerz");
+        assertThat(nativeEntity).isNotNull();
+
+        final ViewDTO viewDTO = nativeEntity.entity();
+        viewDOTShouldHaveGeneralInformation(viewDTO);
```

Comment:
I would split this up into three tests so that it becomes easier to see what's wrong when a test fails. Nice job extracting the assertions though! :+1: